contracts/src/PBHVerifier.sol

```diff
@@ -1,32 +1,176 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
-import {WorldIDProxy} from "@world-id-contracts/abstract/WorldIDProxy.sol";
+import {ByteHasher} from "./helpers/ByteHasher.sol";
+import {PBHExternalNullifier} from "./helpers/PBHExternalNullifier.sol";
+import {IWorldIDGroups} from "@world-id-contracts/interfaces/IWorldIDGroups.sol";
+import {WorldIDImpl} from "@world-id-contracts/abstract/WorldIDImpl.sol";
+import {IEntryPoint} from "@account-abstraction/contracts/interfaces/IEntryPoint.sol";
+import {IPBHVerifier} from "./interfaces/IPBHVerifier.sol";
+import "@BokkyPooBahsDateTimeLibrary/BokkyPooBahsDateTimeLibrary.sol";
 
-/// @title PBH Verifier
+/// @title PBH Verifier Implementation Version 1
 /// @author Worldcoin
-/// @notice An implementation of an on chain PBH verifier.
-/// TODO: Inherit directly from ERC1967Proxy
-contract PBHVerifier is WorldIDProxy {
+/// @notice An implementation of a priority blockspace for humans (PBH) transaction verifier.
+/// @dev This is the implementation delegated to by a proxy.
+contract PBHVerifier is IPBHVerifier, WorldIDImpl {
```

Comment:
Does this need to be a proxy implementation? I think we can deploy this without a proxy.

Reply:
Can you clarify what you mean here? Do you mean remove the `ERC1967Proxy` altogether?